saleor/graphql/discount/mutations/voucher/voucher_code_bulk_delete.py

```diff
@@ -0,0 +1,45 @@
+import graphene
+
+from .....discount import models
+from .....permission.enums import DiscountPermissions
+from .....webhook.event_types import WebhookEventAsyncType
+from ....core import ResolveInfo
+from ....core.descriptions import ADDED_IN_318
+from ....core.mutations import ModelBulkDeleteMutation
+from ....core.types import DiscountError, NonNullList
+from ....core.utils import WebhookEventInfo
+from ....plugins.dataloaders import get_plugin_manager_promise
+from ...types import VoucherCode
+
+
+class VoucherCodeBulkDelete(ModelBulkDeleteMutation):
+    class Arguments:
+        ids = NonNullList(
+            graphene.ID,
+            required=True,
+            description="List of voucher codes IDs to delete.",
+        )
+
+    class Meta:
+        description = "Deletes voucher codes." + ADDED_IN_318
+        model = models.VoucherCode
+        object_type = VoucherCode
+        permissions = (DiscountPermissions.MANAGE_DISCOUNTS,)
+        error_type_class = DiscountError
```

Comment:
Some time ago we agreed to use mutation specific error class so we should create `VoucherCodeBulkDeleteError` and use here